src/components/Calculator.js

```diff
@@ -14,38 +17,47 @@ function Calculator() {
-        <button type="button" className="operators" name="-" onClick={handleButtonChange}>-</button>
-        <button type="button" className="numbers" name="1" onClick={handleButtonChange}>1</button>
-        <button type="button" className="numbers" name="2" onClick={handleButtonChange}>2</button>
-        <button type="button" className="numbers" name="3" onClick={handleButtonChange}>3</button>
-        <button type="button" className="operators" name="+" onClick={handleButtonChange}>+</button>
-        <button type="button" className="numbers span-zero" name="0" onClick={handleButtonChange}>0</button>
-        <button type="button" className="numbers" name="." onClick={handleButtonChange}>.</button>
-        <button type="button" className="operators" name="=" onClick={handleButtonChange}>=</button>
-      </div>
-    </section>
+              <button type="button" className="numbers" name="AC" onClick={handleButtonChange}>AC</button>
+              <button type="button" className="numbers" name="+/-" onClick={handleButtonChange}>+/-</button>
+              <button type="button" className="numbers" name="%" onClick={handleButtonChange}>%</button>
+              <button type="button" className="operators" name="&divide;" onClick={handleButtonChange}>&divide;</button>
+              <button type="button" className="numbers" name="7" onClick={handleButtonChange}>7</button>
+              <button type="button" className="numbers" name="8" onClick={handleButtonChange}>8</button>
+              <button type="button" className="numbers" name="9" onClick={handleButtonChange}>9</button>
+              <button type="button" className="operators" name="x" onClick={handleButtonChange}>x</button>
+              <button type="button" className="numbers" name="4" onClick={handleButtonChange}>4</button>
+              <button type="button" className="numbers" name="5" onClick={handleButtonChange}>5</button>
+              <button type="button" className="numbers" name="6" onClick={handleButtonChange}>6</button>
+              <button type="button" className="operators" name="-" onClick={handleButtonChange}>-</button>
+              <button type="button" className="numbers" name="1" onClick={handleButtonChange}>1</button>
+              <button type="button" className="numbers" name="2" onClick={handleButtonChange}>2</button>
+              <button type="button" className="numbers" name="3" onClick={handleButtonChange}>3</button>
+              <button type="button" className="operators" name="+" onClick={handleButtonChange}>+</button>
+              <button type="button" className="numbers span-zero" name="0" onClick={handleButtonChange}>0</button>
+              <button type="button" className="numbers" name="." onClick={handleButtonChange}>.</button>
+              <button type="button" className="operators" name="=" onClick={handleButtonChange}>=</button>
+            </div>
+          </Col>
```

Comment:
- Hey, great job with your design, however, let's make some adjustments to our calculator as, it's having some extra spaces, and also, the buttons are going out of the container, which is not a good user experience and it doesn't match the template design because of this issue, please, let's work on this. see the images below:



*template design*
![Screenshot from 2022-12-07 02-04-17](https://user-images.githubusercontent.com/59397159/206062944-d3f166a1-5d5f-4b7c-be54-40bf81fa1feb.png)


*your design with button going out the container*
![Screenshot from 2022-12-07 02-02-50](https://user-images.githubusercontent.com/59397159/206062950-12ff7a08-613a-4d2c-9a6e-bb8929889e28.png)


*your design with the extra space*
![Screenshot from 2022-12-07 02-01-20](https://user-images.githubusercontent.com/59397159/206062953-9265d165-e944-42f6-bcde-1c5bcaf8591c.png)